cobalt/BUILD.gn

```diff
@@ -23,7 +24,7 @@ group("gn_all") {
     "//starboard/nplb",
   ]
   if (!is_android) {
-    deps += [ ":cobalt" ]
+    deps += [ ":strip_cobalt_binary" ]
```

Comment:
Could this be a dep of the `cobalt` target instead?

Reply:
That wouldn't make sense, strip_cobalt_binary needs to use the `cobalt` output to run